lean4-infoview/src/infoview/tooltips.tsx

```diff
@@ -70,11 +70,14 @@ export const Tooltip = forwardAndUseRef<HTMLDivElement,
   return ReactDOM.createPortal(popper, document.body)
 })
 
-/** An element which gets highlighted when its DOM children are hovered over. It is implemented
+export type HoverState = 'off' | 'over' | 'ctrlOver'
```

Comment:
It all works, but I'm a little confused about how the code never has to set the HoverState to 'off' except in the intitalState of the React.useState...